gordo_components/server/server.py

```diff
@@ -1,8 +1,16 @@
 # -*- coding: utf-8 -*-
+"""
+This module contains code for generating the Gordo server Flask application.
+
+Running this module will run the application using Flask's development webserver.
+Gunicorn can be used to run the application as AsyncIO workers, having build the Flask
+app with :func:`~gordo_components.server.build_aioapp`.
```

Comment:
Is this a real location? `build_aioapp`?

Reply:
whoops... nope, not any more. Thanks!